src/OrchardCore.Modules/OrchardCore.Feeds/Controllers/FeedController.cs

```diff
@@ -65,23 +65,23 @@ public async Task<ActionResult> Index(string format)
             }
 
             var document = await context.Builder.ProcessAsync(context, async () =>
-           {
-               await bestQueryMatch.FeedQuery.ExecuteAsync(context);
-
-               await _feedItemBuilder.PopulateAsync(context);
-
-               foreach (var contextualizer in context.Response.Contextualizers)
-               {
-                   if (ControllerContext != null)
-                   {
-                       contextualizer(new ContextualizeContext
-                       {
-                           ServiceProvider = _serviceProvider,
-                           Url = Url
-                       });
-                   }
-               }
-           });
+            {
+                await bestQueryMatch.FeedQuery.ExecuteAsync(context);
+            
+                await _feedItemBuilder.PopulateAsync(context);
+            
+                foreach (var contextualizer in context.Response.Contextualizers)
+                {
+                    if (ControllerContext != null)
+                    {
+                        contextualizer(new ContextualizeContext
+                        {
+                            ServiceProvider = _serviceProvider,
+                            Url = Url
+                        });
+                    }
+                }
+            });
```

Comment:
I don't find anything change in this block other than adding unwanted spaces

Reply:
There was a missing single space for the `document` block, I did it for a reason, but seems 70 and 72 has extra tabs